Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -304,6 +301,7 @@ mode would look as follows:
         --set global.tunnel=disabled \\
         --set global.autoDirectNodeRoutes=true \\
         --set global.kubeProxyReplacement=strict \\
+        --set global.nodePort.mode=hybrid \\
```

Comment:
I didn't add it here since it's default when tunneling is disabled. Why we still need it?